external/versions.cmake

```diff
@@ -3,8 +3,8 @@
 
 set(SEQUANT_TRACKED_VGCMAKEKIT_TAG d6746098e63deab4032309c4455bb084a17ff51a)
 
-set(SEQUANT_TRACKED_BOOST_VERSION 1.68)
-set(SEQUANT_TRACKED_BOOST_PREVIOUS_VERSION 1.68)
+set(SEQUANT_TRACKED_BOOST_VERSION 1.72)
+set(SEQUANT_TRACKED_BOOST_PREVIOUS_VERSION 1.72)
```

Comment:
Not sure whether this might have to stay at 1.68? What's the workflow with this previous-version thingy?